core/driver/sources/src/main/scala/org/virtuslab/ideprobe/dependencies/JbrResolvers.scala

```diff
@@ -11,19 +11,22 @@ import org.virtuslab.ideprobe.dependencies.Dependency.Missing
 import org.virtuslab.ideprobe.Extensions.PathExtension
 
 object JbrResolvers {
-  val official =
-    JbrPatternResolver(
-      "https://cache-redirector.jetbrains.com/intellij-jbr/jbr_dcevm-[major]-[platform]-x64-b[minor].tar.gz"
-    )
+  // The `jbr_dcevm` pattern applies only to versions of IntelliJ older than 2022.2 release, whereas
+  // `jbr` pattern applies to newer versions and has dcevm bundled by default.
+  private val officialJbrVersionPatterns = Seq(
```

Comment:
Pls open a PR in git-machete-intellij-plugin that would use this piece in ideprobe.conf... in CircleCI config, use:

```yaml
name: Install sbt
# language=sh
command: |
  apt update
  apt-get install apt-transport-https gnupg -yqq
  echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | tee /etc/apt/sources.list.d/sbt.list
  echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | tee /etc/apt/sources.list.d/sbt_old.list
  curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/scalasbt-release.gpg --import
  chmod 644 /etc/apt/trusted.gpg.d/scalasbt-release.gpg
  apt-get update
  apt-get install sbt
```

and

```yaml
- run:
    name: Build ide-probe fork
    # language=sh
    command: |
      cd ..
      git clone https://github.com/azdrojowa123/ide-probe.git -b fix/jbr_installation_directory
      cd ide-probe/
      sbt api_2_13/publishM2 driver_2_13/publishM2 junit-driver_2_13/publishM2 robot-driver_2_13/publishM2 probe-plugin_2_13/publishM2
```

Reply:
Ok, but I still do not understand fully - after running
`sbt api_2_13/publishM2 driver_2_13/publishM2 junit-driver_2_13/publishM2 robot-driver_2_13/publishM2 probe-plugin_2_13/publishM2`
We have to additionally:
- `grep` published `SNAPSHOT` version number
and
- replace official release in `gradle/libs.versions.toml`
So these two steps have to be created in `.circleci.config.yml` right? And this whole Pull Request would be just for a temporary test - will not be merged? 
Maybe you can just let me know, if there was such a PR in the past and add a link to it? I did not manage to find an example PR unfortunately.

Reply:
> replace official release with new snapshot version in gradle/libs.versions.toml

This is to be done manually by the developer on the PR itself... as they will already know the exact version of ide-probe to use.

> And this whole Pull Request would be just for a temporary test - will not be merged?

Yeah, we generally only want to merge the PR once ide-probe is released... and then use a regular (non-snapshot) ide-probe version. Or rather, this can indeed be a new PR, where ideprobe.conf is adequately updated, and ide-probe version bumped to the released one.

> Maybe you can just let me know, if there was such a PR in the past and add a link to it? I did not manage to find an example PR unfortunately.

Lemme find such commit from reflogs and push as a branch...

Reply:
Okay, just check the commit (`f9ac2f5dc131b2a492a3225be1c58d79758c7baf`) pointed by `demo/ide-probe-snapshot` branch in git-machete-intellij-plugin... that's basically just changes to CircleCI config + ide-probe version change in the TOML catalog

Reply:
Success: https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1080